src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -208,7 +208,7 @@ const DiscoverySearch = ({
             onPress={handleOnClearText}
             style={styles.clearIconContainer}
             android_ripple={{
-              color: COLORS.lightgrey,
+              color: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ripple color for the `android_ripple` prop has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Make sure this color change is consistent with the overall app theme and provides a good user experience in terms of visibility and aesthetics.
```diff
-    color: COLORS.lightgrey,
+    color: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->